assets/javascripts/utils/cookie.js

```diff
@@ -15,7 +15,15 @@ define(['utils/base64'], function (base64) {
             expires = '';
         }
 
-        document.cookie = [name, '=', value, expires, '; path=/', secureCookieString ].join('');
+        var domain = trimSubdomain ? '; domain=' + getShortDomain() : '';
```

Comment:
Can this be a `const`?` Can also use template strings for bonus points!

Reply:
it will be the only const in the whole codebase so I just tend to match the existing style. Same with template strings. 
It's tricky, I guess we could just start converting as we go but then PRs become a mish mash of actual functional changes and stylistic changes.

Reply:
meep https://github.com/guardian/subscriptions-frontend/pull/1177/files#diff-38cbb81a25ab7304ca9c2b69638a79d0R25